k8s/Makefile

```diff
@@ -94,29 +94,50 @@ k8s-delete-ns:
 k8s-services:
 	j2 app.svc.yaml.j2 | ${KC} apply -f -
 
+test-k8s-services:
+	j2 app.svc.yaml.j2 | kubeval --strict --ignore-missing-schemas
+
```

Comment:
Is this addition separate from what `test-k8s-deployments` uses? Should it be included in there, too?

Reply:
Apologies not quite following this here :)